package.json

```diff
@@ -5,7 +5,7 @@
     "type": "git",
     "url": "https://github.com/getsentry/sentry-capacitor"
   },
-  "version": "2.1.0",
+  "version": "2.1.3",
```

Comment:
Q: Is the version bump temporary for testing purposes?

Reply:
thanks for catching this, it was a last minute change because ionic wasn't catching the right version, despite being on node modules